homeassistant/components/wmspro/cover.py

```diff
@@ -99,3 +120,66 @@ class WebControlProRollerShutter(WebControlProCover):
     _drive_action_desc = (
         WMS_WebControl_pro_API_actionDescription.RollerShutterBlindDrive
     )
+
+
+class WebControlProSlatDrive(WebControlProCover):
+    """Representation of a WMS based blind using a slat drive."""
+
+    _attr_device_class = CoverDeviceClass.BLIND
+    _drive_action_desc = WMS_WebControl_pro_API_actionDescription.SlatDrive
+
+
+class WebControlProSlatRotate(WebControlProCover):
+    """Representation of a WMS based blind using only a slat rotate."""
+
+    _attr_device_class = CoverDeviceClass.BLIND
+    _drive_action_desc = WMS_WebControl_pro_API_actionDescription.SlatRotate
+    _drive_action_attr = "rotation"
+
+
+class WebControlProSlatDriveRotate(WebControlProSlatDrive):
+    """Representation of a WMS based blind which supports tilting."""
+
+    _tilt_action_desc = WMS_WebControl_pro_API_actionDescription.SlatRotate
+    _tilt_action_attr = "rotation"
+
+    async def async_open_cover(self, **kwargs: Any) -> None:
+        """Open the cover and tilt like the hub."""
+        await super().async_open_cover(**kwargs)
+        await self.async_open_cover_tilt(**kwargs)
+
+    async def async_close_cover(self, **kwargs: Any) -> None:
+        """Close the cover and tilt like the hub."""
+        await super().async_close_cover(**kwargs)
+        await self.async_close_cover_tilt(**kwargs)
+
+    @property
+    def current_cover_tilt_position(self) -> int | None:
+        """Return current position of cover tilt."""
+        action = self._dest.action(self._tilt_action_desc)
+        return ranged_value_to_percentage(
+            (action.minValue, action.maxValue),
+            action[self._tilt_action_attr],
+        )
+
+    async def async_set_cover_tilt_position(self, **kwargs: Any) -> None:
+        """Set the cover tilt position."""
+        action = self._dest.action(self._tilt_action_desc)
+        rotation = percentage_to_ranged_value(
+            (action.minValue, action.maxValue),
+            kwargs[ATTR_TILT_POSITION],
+        )
+        kwargs = {self._tilt_action_attr: rotation}
+        await action(**kwargs)
+
+    async def async_open_cover_tilt(self, **kwargs: Any) -> None:
+        """Open the cover tilt."""
+        action = self._dest.action(self._tilt_action_desc)
+        kwargs = {self._tilt_action_attr: action.maxValue}
+        await action(**kwargs)
+
+    async def async_close_cover_tilt(self, **kwargs: Any) -> None:
+        """Close the cover tilt."""
+        action = self._dest.action(self._tilt_action_desc)
+        kwargs = {self._tilt_action_attr: action.minValue}
```

Comment:
@stefannikolei can you please comment if the minValue corresponds to opening the tilt?